README.md

```diff
@@ -129,34 +129,56 @@ No | Irrelevant | No  | No  | `Requisite` and `Preferred` both nil
 
 ### Capacity support
 
-> :warning: *Warning:* This is an alpha feature and only supported by
-> Kubernetes >= 1.19 if the `CSIStorageCapacity` feature gate is
-> enabled.
-
 The external-provisioner can be used to create CSIStorageCapacity
 objects that hold information about the storage capacity available
 through the driver. The Kubernetes scheduler then [uses that
 information](https://kubernetes.io/docs/concepts/storage/storage-capacity)
 when selecting nodes for pods with unbound volumes that wait for the
 first consumer.
 
-Currently, all CSIStorageCapacity objects created by an instance of
-the external-provisioner must have the same
-[owner](https://kubernetes.io/docs/concepts/workloads/controllers/garbage-collection/#owners-and-dependents). That
-owner is how external-provisioner distinguishes between objects that
-it must manage and those that it must leave alone. The owner is
-determine with the `POD_NAME/POD_NAMESPACE` environment variables and
-the `--capacity-ownerref-level` parameter. Other solutions will be
-added in the future.
+All CSIStorageCapacity objects created by an instance of
+the external-provisioner have certain labels:
+
+  * `csi.storage.k8s.io/drivername`: the CSI driver name
+  * `csi.storage.k8s.io/managed-by`: external-provisioner for central
+    provisioning, external-provisioner-<node name> for distributed
+    provisioning
+
+They get created in the namespace identified with the `NAMESPACE`
+environment variable.
+
+Each external-provisioner instance manages exactly those objects with
+the labels that correspond to the instance.
+
+Optionally, all CSIStorageCapacity objects created by an instance of
+the external-provisioner can have an
+[owner](https://kubernetes.io/docs/concepts/workloads/controllers/garbage-collection/#owners-and-dependents).
+This ensures that the objects get removed automatically when uninstalling
```

Comment:
Added.

As you said, orphaned and incorrect objects cause no permanent damage for pods with normal PVCs (which includes generic ephemeral volumes), so it's not too bad. It's worse for CSI ephemeral volumes because then a pod might get scheduled onto a node where the driver then cannot provide the volume and the pod is stuck there (no rescheduling). I think that was what we chatted about, not the scheduler itself hanging.

Reply:
Ah yeah. Could you maybe mention this in the design doc somewhere (if it's not there already)?

Reply:
It is covered by the generic ephemeral volume KEP.